File: java/oop/AccountPolymorphismApp/src/com/techlabs/account/Account.java
package com.techlabs.account;

public abstract class Account {
	private final int acno;
	private final String acnt_Name;
	protected double balance=0;
	public Account(int acno,String  acntname,double balance){
		this.acno=acno;
		this.acnt_Name=acntname;
		this.balance=balance;
	}
	public int getAcntNo(){
		return this.acno;
	}
	public String getAcntName(){
		return this.acnt_Name;
	}
	public double getBalance(){
		return this.balance;
	}
	
	public void depositeAmount(double amt){
		this.balance+=amt;
	}
	public abstract void withdraw(double amt);
}

File: dotNet/csharp/AccountPolymorphismApp/AccountPolymorphismApp/Account.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AccountPolymorphismApp
{
   public abstract class Account
    {
        private  int _acno;
        private  string _acntName;
        protected double _balance;
        public Account(int acno, String acntname, double balance)
        {
            this._acno = acno;
            this._acntName = acntname;
            this._balance = balance;
        }

        public string Name
        {
            get
            {
                return this._acntName;
            }
        }

        public int AcntNo
        {
            get
            {
                return _acno;
            }
        }

        public double Balance
        {
            get
            {
                return _balance;
            }
        }

        public override string ToString()
        {
            return "Account Holder :" + this.Name
                + "\nAccount No :" + this.AcntNo
                + "\nBalance :" + this.Balance;
        }
        public void DepositeAmount(double amt)
        {
            this._balance += amt;
        }
        public abstract void Withdraw(double amt);

    }
}
